### PR TITLE
Wrote command-line script to use the library.

### PR DESCRIPTION
**Pull request by ypid**

### CODE REVIEW

**Comment by schmijos** [comment]
Cool one. Thx!

Would you mind changing the file extension into `.py` or is there a special reason for that you left it away?

Also could please conform to at least [PEP8](https://pypi.python.org/pypi/pep8)?

File: html_table_to_csv_converter [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+# published by the Free Software Foundation, version 3 of the
+# License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
```
Comment (schmijos) [inline html_table_to_csv_converter]: Would you mind to remove this license header? You can instead change the LICENSE of the whole project into AGPL if you want.

File: html_table_to_csv_converter [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                '.'.join(
+                    (
+                        args.output,
+                        str(index),
+                        args.output_format)
+                    ),
+                tables[index]
+            )
+    else:
+        pprint(table)
```
Comment (schmijos) [inline html_table_to_csv_converter]: This should be `pprint(tables)`.

**Comment by ypid** [comment]
Fixed. Thanks. There is a reason why the script is not called `html_table_to_csv_converter.py` and that is because it is a command line script and it is uncommon to execute programs which are in your `$PATH` with *.py …

**Pull request merged**